### ISSUE TITLE
line for line identical 

### ISSUE THREAD

**Issue by paulrudy** [schmidlorenz/alfred-ddgnext#1]
I noticed your workflow on Packal. It's interesting that it's character for character identical to the workflow I posted here in July 2014, adapted from the same people you attribute:

http://www.alfredforum.com/topic/4354-search-duckduckgo-next-with-google-suggest-completions/?p=28451

Just a coincidence, or did you copy my adapted workflow?

**Comment by lorenzschmid** [comment]
I'm really sorry to hear that and glad you reached out to me. I was not aware of your adaptation and definitely did not copy your code (there are slight differences if you look closely). I contacted @poritsky per email last week to ask him about adapting and publishing his code which I found on his [website](http://www.candlerblog.com/2014/05/08/duckduckgo-next/) - I was not aware of the separate forum post.

To be honest, it is less important to me if I am attributed with the code but rather that someone puts it on [Packal](http://www.packal.org/workflow/duckduckgo-auto-suggestion) and that the source is on GitHub so it gets further improved and bugs can be collected.

I acknowledge that you were first putting your adaptation online but insist that the code was not copied - being rather new to the opensource community I do not know how to proceed now and am open for suggestions.
